app/controllers/pageflow/editor/files_controller.rb

```diff
@@ -54,7 +54,7 @@ def file_type
       def file_params
         params.require(file_type.param_key)
           .permit(:attachment => [:tmp_path, :original_name, :content_type])
-          .merge(params.require(file_type.param_key).permit(:attachment))
+          .merge(params.require(file_type.param_key).permit(:attachment, :parent_id, :parent_type))
```

Comment:
We need a controller spec that ensures associating a parent file works. I think the attribute names should be `parent_file` etc here. Let's also add a test for the case of a parent file which does not support the given file as nested file, so we see that the validation is applied correctly.